content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -173,38 +174,45 @@ The following example shows accessing a service running in the Kubernetes cluste
 1.  First, on the cluster admin machine get the virtual IP address (`clusterIP`) for the service:
 
     {{< text bash >}}
-$ kubectl -n bookinfo get svc productpage -o jsonpath='{.spec.clusterIP}'
+$ kubectl get svc productpage -o jsonpath='{.spec.clusterIP}'
 10.55.246.247
     {{< /text >}}
 
 1.  Then on the mesh expansion machine, add the service name and address to its `etc/hosts` file. You can then connect to
     the cluster service from the VM, as in the example below:
 
     {{< text bash >}}
-$ sudo echo "10.55.246.247 productpage.bookinfo.svc.cluster.local" >> /etc/hosts
-$ curl productpage.bookinfo.svc.cluster.local:9080
+$ sudo echo "10.55.246.247 productpage.default.svc.cluster.local" >> /etc/hosts
+$ curl productpage.default.svc.cluster.local:9080
 ... html content ...
     {{< /text >}}
 
 ## Running services on a mesh expansion machine
 
-You add VM services to the mesh by configuring a
+1. We first setup a HTTP server on the VM instance, serving HTTP traffic on port 8080.
+
+    {{< text bash >}}
+    $ gcloud compute ssh your-vm-instance
+    $ python -m SimpleHTTPServer 8080
+    {{< /text >}}
+
+1. Configure Service Discovery for VM by `ServiceEntry`. You add VM services to the mesh by configuring a
 [`ServiceEntry`](/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry). A service entry lets you manually add
 additional services to Istio's model of the mesh so that other services can find and direct traffic to them. Each
 `ServiceEntry` configuration contains the IP addresses, ports, and labels (where appropriate) of all VMs exposing a
 particular service, as in the following example.
 
 {{< text bash yaml >}}
-$ kubectl -n test apply -f - <<EOF
```

Comment:
Why the removal of the namespace? And what isn't this object being created in `${SERVICE_NAMESPACE}`?

Reply:
I personally dont think namespace are relevant here  using default is fine. But changed to `SERVICE_NAMESPACE` for consistency.

Reply:
I think it is helpful to use the namespace consistently throughout this doc since that will be representative of how an actual user is going to use this doc; we don't expect users to stuff everything in the default namespace, so why not show them how to use a custom namespace?

Reply:
I'm okay to use a non default namespace.

But that adds up more complexity for this tutorial setup. Customer needs to create the namespace, create istio.default service account and verify Citadel catches up generate corresponding key/cert. This makes the tutorial more fragile and complicated.

I think the purpose of the doc is more of illustration rather the real usage in prod. editing `/etc/hosts/` manual download deb package, etc. So robustness and simplicity is more important than being real.